drake/solvers/JointSpaceMotionPlanningTree.m

```diff
@@ -44,13 +44,13 @@
       obj.q_nom = q;
     end
 
-    function [q, valid] = solveIK(obj, q_seed, q_nom, additional_constraints)
+    function [q, valid, infeasibleConstraints] = solveIK(obj, q_seed, q_nom, additional_constraints)
```

Comment:
Is the third output here being used? Also, to conform with Drake's capitalization conventions it should be `infeasible_constraints`